awx/main/tests/factories/fixtures.py

```diff
@@ -141,11 +141,11 @@ def mk_job(job_type='run', status='new', job_template=None, inventory=None,
 
```

Comment:
Almost everything past this point is

`s/credential/credentials`

...in the tests.